sailthru_content/services/tests/test_sailthru_translation_service.py

```diff
@@ -103,8 +104,13 @@ def _get_expected_sailthru_item(self, seat_type=None):
             'date': '2016-09-23'
         }
 
+        if seat_type in ['professional', 'verified']:
```

Comment:
You can just replace this list with the `sailthru_translation_service.REQUIRED_SEATS_TYPES`